tests/integration/context_variables_and_function_types/resources/tmc/dataview.pml

```diff
@@ -0,0 +1,152 @@
+#define Worker_States int
+#define Worker_Context_state Worker_States
+#define Worker_Context_init_done bool
+#define Worker_Context_check_param MyInteger
+#define Worker_Context_result_param MyInteger
+#define Controller_States int
+#define Controller_Context_state Controller_States
+#define Controller_Context_init_done bool
+#define Controller_Context_check_param MyInteger
+#define Controller_Context_result_param MyInteger
+#define Controller_Context_error MyInteger
+#define MyInteger int
+#define T_Int32 int
+#define T_UInt32 int
+#define T_Int8 int
+#define T_UInt8 int
+#define T_Boolean bool
+#define Context_worker_coeff MyInteger
+#define Worker_States_wait 0
+#define Controller_States_wait 0
+typedef Context_worker {
+    Context_worker_coeff coeff;
+}
+
+typedef Controller_Context {
+    Controller_Context_state state;
+    Controller_Context_init_done init_done;
+    Controller_Context_check_param check_param;
+    Controller_Context_result_param result_param;
+    Controller_Context_error error;
+}
+
+typedef T_Null_Record {
+    bit dummy;
+}
+
+typedef Worker_Context {
+    Worker_Context_state state;
+    Worker_Context_init_done init_done;
+    Worker_Context_check_param check_param;
+    Worker_Context_result_param result_param;
+}
+
+Context_worker actuator_ctxt;
+Context_worker worker_ctxt;
```

Comment:
I assume that this is a byproduct of brute-force ASN.1 translation.... as worker itself is not a part of the system state, right?

Reply:
Brute-translated from ASN.1